backend/src/main/resources/application.properties

```diff
@@ -42,11 +42,13 @@ login-attempts.max-attempts-per-interval=5
 #   JWT
 jwt.access-expiration-ms=${JWT_ACCESS_EXPIRATION}
 jwt.email-expiration-ms=${EMAIL_EXPIRATION}
+jwt.refresh-expiration-ms=${JWT_REFRESH_EXPIRATION}
+jwt.registration-invite-expiration-ms=${JWT_REGISTRATION_INVITE_EXPIRATION}
 jwt.secret=${JWT_SECRET}
-jwt.refresh-expiration-ms=1800000000
 #   App Properties - Email Jwt Verification Token
 backend.app.emailConfirmationLink=http://${SKYNEST_HOST}:${FE_PORT}/login?token=
 backend.app.passwordResetLink=http://${SKYNEST_HOST}:${FE_PORT}/confirm-password?token=
+backend.app.registrationInviteLink=http://${SKYNEST_HOST}:${FE_PORT}/register?token=
```

Comment:
this PR only includes sending the email?

Reply:
Yeah, there will be 1 more for changing registration